dir.props

```diff
@@ -48,6 +48,12 @@
     <ToolsDir Condition="'$(UseToolRuntimeForToolsDir)'=='true'">$(ToolRuntimePath)</ToolsDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)Tools/</ToolsDir>
     <ToolRuntimePath Condition="'$(ToolRuntimePath)'==''">$(ToolsDir)</ToolRuntimePath>
+    
+    <!-- Temporarily enable local build of tools -->
```

Comment:
Please provide a suggested location.  Note this will still need to change to import whatever props you're suggesting I move these too.